packages/realm-server/tests/indexing-test.ts

```diff
@@ -1221,16 +1221,48 @@ module(basename(__filename), function () {
               return this.author?.firstName + '-poo';
             }
           })
-          static embedded = class Embedded extends Component<typeof this> {
-            <template><@fields.firstName/> (<@fields.nickName/>)</template>
-          }
-          static fitted = class Fitted extends Component<typeof this> {
-            <template><@fields.firstName/> (<@fields.nickName/>)</template>
-          }
```

Comment:
why did we do this? this breaks the resulting search in this test because these items won't be in the search doc since they are not longer in these templates